### ISSUE TITLE
Schema name in Postgres `from` is incorrectly quoted

### ISSUE THREAD

**Issue by mklopets** [PRQL/prql#822]
This behaviour was introduced in https://github.com/prql/prql/pull/809

Assume you have a Postgres database with a schema `some_schema` (this could even be `public`)
```
prql dialect:postgres
from some_schema.tablename
```

compiles to
```
SELECT
  some_schema.tablename.*
FROM
  "some_schema.tablename"
```

which is an invalid query – the schema name shouldn't be inside quotes (postgres will complain with `relation "some_schema.tablename" does not exist`.

If we want to keep the quotes, a valid Postgres query would be:

```diff
-  "some_schema.tablename"
+  "some_schema"."tablename"
```

or just

```diff
-  "some_schema.tablename"
+  some_schema.tablename
```

**Comment by max-sixty** [comment]
Ah, this is unfortunate, #809 was quite elegant. We can make it split by `.` I think.

Does this block you? We can revert #809 if it does (we have a [policy](https://github.com/prql/prql/blob/main/CONTRIBUTING.md) of fast merges and fast reverts!)

**Comment by mklopets** [comment]
it's non-blocking until prql-js has a new version we need to update to for other reasons (the latest published version, `v0.2.2` is fine)

splitting by `.` would be neat, though I'm not sure which other dialects this applies to

**Comment by max-sixty** [comment]
OK. Probably we should revert rather than block a release, but let me see whether it's an easy fix

**Comment by blackary** [comment]
Similar issue with BigQuery

```
prql dialect:bigquery
from db.schema.table
``` 
-> 

```sql
SELECT
  db.schema.table.*
FROM
  `db.schema.table`
 ```
 
 Which causes this error:
 `Unrecognized name: db at [2:3]`
 
 In this case the backticks after the FROM are fine, but they are missing after the SELECT
 
 ```diff
 -  db.schema.table.*
+  `db.schma.table`.*
```

**Comment by huw** [comment]
This is also breaking Spark SQL (not a supported dialect, I know)—in this case, the `FROM` clause needs a table name of either `some_schema.tablename` or 
```
`some_schema`.`tablename`
```
(Seems like it would be ideal to split on `.` in all dialects, regardless of quote marks).

**Comment by max-sixty** [comment]
Closed by #838. Thanks for everyone's patience
